docs/pages/docs/guides/images-and-files.mdx

```diff
@@ -0,0 +1,266 @@
+   S3_REGION: region = 'ap-southeast-2',
+   // The Access Key ID and Secret that has read/write access to the S3 bucket
+   S3_ACCESS_KEY_ID: accessKeyId = 'keystone',
+   S3_SECRET_ACCESS_KEY: secretAccessKey = 'keystone',
+   // The base URL to serve assets from
+   ASSET_BASE_URL: baseUrl = 'http://localhost:3000,
+ } = process.env;
+
+export default config({
+  db: {
+    provider: 'sqlite',
+    url: process.env.DATABASE_URL || 'file:./keystone-example.db',
+  },
+  lists,
+  storage: {
+    // The key here will be what is referenced in the image field
+    my_local_images: {
+      // Images that use this store will be stored on the local machine
+      kind: 'local',
+      // This store is used for the image field type
+      type: 'image',
+      // The URL that is returned in the Keystone GraphQL API
+      generateUrl: path => `${baseUrl}/images${path}`,
+      // The route that will be created in Keystone's backend to serve the images
+      serverRoute: {
+        path: '/images',
+      },
+      storagePath: 'public/images',
+    },
+    // The key here will be what is referenced in the file field
+    my_s3_files: {
+      // Files that use this store will be stored in an s3 bucket
+      kind: 's3',
+      // This store is used for the file field type
+      type: 'file',
+      // The S3 bucket name pulled from the S3_BUCKET_NAME environment variable above
+      bucketName,
+      // The S3 bucket region pulled from the S3_REGION environment variable above
+      region,
+      // The S3 Access Key ID pulled from the S3_ACCESS_KEY_ID environment variable above
+      accessKeyId,
+      // The S3 Secret pulled from the S3_SECRET_ACCESS_KEY environment variable above
+      secretAccessKey
+      // The S3 links will be signed so they remain private
+      signed: { expiry: 5000 },
+    },
+  },
+});
+```
+
+!> You can define as many stores as required for your use case
+
+## Field types - `image` and `file`
+
+Once you have your `storage` configuration in Keystone you can then use the `image` and `file` [field types](../apis/fields)
+
+Within an existing [list](../apis/schema) use the `image` of `file` field as follows:
+
+```typescript
+lists: {
+    user: list({
+        fields: {
+            /** other fields */
+            avatar: image({ storage: 'my_local_images' }),
+            someFile: file({ storage: 'my_s3_files' }),
+        }
+    })
+}
+```
+
+!> The `storage` name here relates to the `storage` defined in your `keystone.ts`
+
+## Relationships - ie Image Gallery
+
+If you want your images (and files) to be reused and accessible across multiple difference lists or in a [Document Field](./document-fields). You can setup a Gallery list and use relationships to reference this.
+
+For example you might have a schema that looks like this:
+
+```typescript
+// schema.ts
+import { list } from '@keystone-6/core';
+import { text, image } from '@keystone-6/core/fields';
+
+export const lists = {
+    Image: list({
```

Comment:
not sure the tab/indent approach is consistent with other guides (it looks larger than I recall)

might be worth a doublecheck 🙂